rpm/0001-Add-sb2install-support-to-osc.patch

```diff
@@ -1,40 +1,37 @@
-From a9c30ba58797688d3e5de1ffb50bdb4015e4fcbb Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Marko Lemmetty <marko.lemmetty@jollamobile.com>
 Date: Fri, 15 May 2020 13:46:20 +0300
-Subject: [PATCH 1/8] Add sb2install support to osc
+Subject: [PATCH] Add sb2install support to osc
 
 ---
  osc/build.py | 3 +++
  1 file changed, 3 insertions(+)
 
 diff --git a/osc/build.py b/osc/build.py
-index 8f599b8..dfa9997 100644
+index c47f6b2c..d2bdb888 100644
 --- a/osc/build.py
 +++ b/osc/build.py
-@@ -158,6 +158,7 @@ class Buildinfo:
-             self.pathes.append(node.get('project')+"/"+node.get('repository'))
+@@ -197,6 +197,7 @@ class Buildinfo:
+                 self.urls[node.get('project')+"/"+node.get('repository')] = baseurl + '/%(arch)s/%(filename)s'
  
          self.vminstall_list = [ dep.name for dep in self.deps if dep.vminstall ]
 +        self.sb2install_list = [ dep.name for dep in self.deps if dep.sb2install ]
          self.preinstall_list = [ dep.name for dep in self.deps if dep.preinstall ]
          self.runscripts_list = [ dep.name for dep in self.deps if dep.runscripts ]
          self.noinstall_list = [ dep.name for dep in self.deps if dep.noinstall ]
-@@ -195,6 +196,7 @@ class Pac:
-                   'project', 'repository',
```

Comment:
Why added sysroot here?

Reply:
This from the upstream code in the diff mentioned as context:
https://github.com/openSUSE/osc/blob/master/osc/build.py#L234